src/rate/formatRateAsEquation.js

```diff
@@ -0,0 +1,70 @@
+import formatRate from './formatRate';
+import { formatAmount } from '../currency';
+import config from './config';
+import { DEFAULT_RATE_MULTIPLIER } from '../defaults';
+
+export default function(
+  rate,
+  sourceCurrency,
+  targetCurrency,
+  { lhs = 'auto', lhsMultiplier = null } = {},
+) {
+  if (!sourceCurrency || !targetCurrency) {
+    return undefined;
+  }
+
+  let equation = {
+    lhsCurrency: sourceCurrency,
+    lhsValue: 1,
+    rhsCurrency: targetCurrency,
+    rhsValue: rate,
+  };
+
+  equation = invertEquationIfNeeded(equation, lhs, config);
```

Comment:
Optional:

```suggestion
  if (
    reference === 'target' ||
    (reference === 'auto' && shouldAutomaticallyInvert(rate, sourceCurrency, targetCurrency))
  ) { equation = invertEquation(equation) }
```

(Assumes variable rename to `reference`)

Helper functions:

```js

// invertEquationIfNeeded gets removed.

function shouldAutomaticallyInvert(rate, sourceCurrency, targetCurrency) {
  // ignores rate and targetCurrency args for now, just uses config.
  return (config[sourceCurrency] || {}).hasInversionEnabled;
}

function invertEquation(equation) {
  return {
    ...equation,
    lhsCurrency: equation.rhsCurrency,
    rhsCurrency: equation.lhsCurrency,
    rhsValue: 1 / equation.rhsValue
  };
}
```